azavea/pfb-network-connectivity
# Feature/kak/fix fontello#413

## Overview

Load fontello CSS and fonts, and fix apostrophe-like characters in Methodology supplied text so production compiled Angular doesn't replace it with something strange.


### Demo

A fontello icon:
![local_server_fontello](https://cloud.githubusercontent.com/assets/960264/25972623/c221e288-366e-11e7-9d8e-7ad6923fbd83.png)



### Notes

Fiddled with gulp build to try to convince it to compile the Fontello CSS in with the compiled/minified SASS sources. Decided to just copy and load it directly instead, which is simple and works with local watch server and with production build.

Removed an 'other' gulp task that seemed to only serve to create extra, unnecessary copies of misplaced font resources in `dist` directory.


## Testing Instructions

 * Rebuild with `./scripts/update` to get the production static files to update (served on port 9200)
 * Check local watch dev build (port 9301) and prod build (9200)
 * Go to methodology page (link in header)
 * Should have Fontello icon above content heading
 * Should have no weird characters in text (search page for `doesn't` to check)

Closes #413 
Closes #425
